src/models/Commission.js

```diff
@@ -0,0 +1,48 @@
+module.exports = function (sequelize, DataTypes) {
+  const Commission = sequelize.define(
+    'CommissionModel',
+    {
+      commission_id: {
+        type: DataTypes.UUID,
+        defaultValue: DataTypes.UUIDV4,
+        primaryKey: true,
+      },
+      commission_name: DataTypes.TEXT,
+      commission_type: DataTypes.TEXT
```

Comment:
Hola Diego, a que se refiere con comission_type? eso no lo había considerado en la db

Reply:
Faltó aclarar, si revisas este URL: https://www.congreso.gob.pe/CuadrodeComisiones/
Te puedes dar cuenta que en el dropdown Comisión se listan diferentes tipos de comisiones, ese campo es para hacer esa diferencia. La alternativa es crear una tabla como CommissionTypes y crear la relación con la tabla de Comisiones.

Reply:
Dale, no sabía esto, no se si necesitemos crear una tabla adicional, aunque siguiendo la lógica que hemos hecho para los roles, creo que sería la alternativa

Reply:
solved in ca98026